File: test/aidbox_sdk/snapshots/typescripthl7-fhir-r4-core/OrganizationAffiliation.ts
import { Period } from "./Period";
import { CodeableConcept } from "./CodeableConcept";
import { ContactPoint } from "./ContactPoint";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type OrganizationAffiliation = DomainResource & {
    specialty?: CodeableConcept[];
    organization?: Reference;
    participatingOrganization?: Reference;
    active?: boolean;
    code?: CodeableConcept[];
    identifier?: Identifier[];
    telecom?: ContactPoint[];
    network?: Reference[];
    period?: Period;
    location?: Reference[];
    endpoint?: Reference[];
    healthcareService?: Reference[];
};

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/OrganizationAffiliation.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Period
from base import CodeableConcept
from base import ContactPoint
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class OrganizationAffiliation(DomainResource):
    specialty: Optional[List[CodeableConcept]] = field(default_factory=list)
    organization: Optional[Reference] = None
    participating_organization: Optional[Reference] = None
    active: Optional[bool] = None
    code: Optional[List[CodeableConcept]] = field(default_factory=list)
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    telecom: Optional[List[ContactPoint]] = field(default_factory=list)
    network: Optional[List[Reference]] = field(default_factory=list)
    period: Optional[Period] = None
    location: Optional[List[Reference]] = field(default_factory=list)
    endpoint: Optional[List[Reference]] = field(default_factory=list)
    healthcare_service: Optional[List[Reference]] = field(default_factory=list)